appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.validators.js

```diff
@@ -19,7 +29,36 @@ export const validateCommentSubmittedDateFields = createDateInputFieldsValidator
 	'date',
 	'submitted date'
 );
+
 export const validateCommentSubmittedDateValid = createDateInputDateValidityValidator(
 	'date',
 	'submitted date'
 );
+
+export const validateInterestedPartyDetails = createValidator(
+	createTextInputValidator(
+		'firstName',
+		'Enter the first name',
+		250,
```

Comment:
Should these use constants rather than hardcoded values?

Reply:
Char limit now from a constant. Error messages same answer as above.